app/assets/stylesheets/article-show.scss

```diff
@@ -886,26 +868,15 @@ article {
     position: fixed;
     bottom: 0;
     z-index: 22;
-    background: white;
-    border: 1px solid darken($light-medium-gray, 2%);
-    @include themeable(
-      border,
-      theme-container-border,
-      1px solid darken(
-        $light-medium-gray, 2%
-      )
-    );
-    @include themeable(
-      box-shadow,
-      theme-container-box-shadow,
-      $bold-shadow
-    );
     width: 100%;
-    @include themeable(
-      background,
-      theme-background,
-      $tan
-    );
+    @include themeable(background, theme-container-background, $tan);
+    @include themeable(border, theme-container-border, none);
+    @include themeable(box-shadow, theme-container-box-shadow, $bold-shadow);
+
+    @media screen and (min-width: 950px) {
```

Comment:
probably for the future and not for now but it would be cool if we could add our media query breakpoints to variables and define them upfront instead of using the numbers everywhere,

Reply:
100% agree. But for future. Refactoring some bits and pieces of CSS is on my plate, but again - definitely separate PR(s)